### ISSUE TITLE
Code spat out and not reloading browser

### ISSUE THREAD

**Issue by BeardedC0der** [hayes0724/shopify-webpack-themekit#3]
Hey, I'm having an issue where code is spitting out at the bottom of the page, i know this was fixed in the latest slate version but do you know why it's happening on yours?

`");
          } else {
            win._boomrl = function() {
              bootstrap();
            };
            if (win.addEventListener) {
              win.addEventListener("load", win._boomrl, false);
            } else if (win.attachEvent) {
              win.attachEvent("onload", win._boomrl);
            }
          }
          doc.close();
        }
        var link = document.createElement("link");
        if (link.relList &&
          typeof link.relList.supports === "function" &&
          link.relList.supports("preload") &&
          ("as" in link)) {
          window.BOOMR.snippetMethod = "p";
          link.href = window.BOOMR.url;
          link.rel = "preload";
          link.as = "script";
          link.addEventListener("load", promote);
          link.addEventListener("error", function() {
            iframeLoader(true);
          });
          setTimeout(function() {
            if (!promoted) {
              iframeLoader(true);
            }
          }, LOADER_TIMEOUT);
          BOOMR_lstart = new Date().getTime();
          parentNode.appendChild(link);
        } else {
          iframeLoader(false);
        }
        function boomerangSaveLoadTime(e) {
          window.BOOMR_onload = (e && e.timeStamp) || new Date().getTime();
        }
        if (window.addEventListener) {
          window.addEventListener("load", boomerangSaveLoadTime, false);
        } else if (window.attachEvent) {
          window.attachEvent("onload", boomerangSaveLoadTime);
        }
        if (document.addEventListener) {
          document.addEventListener("onBoomerangLoaded", function(e) {
            e.detail.BOOMR.init({
              ResourceTiming: {
                enabled: true,
                trackedResourceTypes: ["script", "img", "css"]
              },
            });
            e.detail.BOOMR.t_end = new Date().getTime();
          });
        } else if (document.attachEvent) {
          document.attachEvent("onpropertychange", function(e) {
            if (!e) e=event;
            if (e.propertyName === "onBoomerangLoaded") {
              e.detail.BOOMR.init({
                ResourceTiming: {
                  enabled: true,
                  trackedResourceTypes: ["script", "img", "css"]
                },
              });
              e.detail.BOOMR.t_end = new Date().getTime();
            }
          });
        }
      })();
    

    if (!isDuplicatedThankYouPageView()) {
      setCookieIfThankYouPage();
      
        window.ShopifyAnalytics.lib.page(
          null,
          {"pageType":"home"}
        );
      
      
    }
  });`

Additionally whenever i make changes browsersync is not working, it seems to be hanging in the terminal.

Thanks

**Comment by hayes0724** [comment]
The code showing in header is a Shopify issue when using development themes. You can fix it by commenting out content_for_header and replacing with this:
```
{% comment %}{{ content_for_header }}{% endcomment %}
{{ content_for_header | replace: "<body onload='document._boomrl();'>", "<bodx onload='document._boomrl();'>" }}
```

For live themes you can use `{{ content_for_header }}`

**Comment by BeardedC0der** [comment]
@hayes0724 thanks. Do you have any idea why files aren't being watched? Browsersync just says `[Browsersync] Reloading Browsers...` in my terminal but nothing happens. Doesn't seem to be uploading any files to shopify either.

Forgot to mention yesterday, this is a great little project you've put together, nice work! I'm trying to move away from slate now that it's been deprecated.

**Comment by hayes0724** [comment]
I pushed this update early, I still need to finish:
1. Browsersymc watching dist instead of root
2. Themekit watch dist instead of root
3. Webpack asset server

Next update will be ready in couple days, until then the last commit eb1f7159e264d9497df147757436db26595abe80 works.

**Comment by hayes0724** [comment]
fixed with 678ae09238d6b193b78837cf5e0cbe45025b66ff
